metadata.cfg

```diff
@@ -44,26 +19,22 @@ classifiers:
         Operating System :: Unix
         Programming Language :: C
         Programming Language :: Python
-        Programming Language :: Python :: 2
-        Programming Language :: Python :: 2.7
```

Comment:
This seems like an issue? Unless we are not planning to update until the Py3 project is done.

Reply:
I think that for our own needs we'll publish the last compatible tags in version branches

Reply:
As I talked with kostas we can release from the tag v.1.4.0 which solves our compatibility issues with debian buster.